cms/static/js/views/utils/create_library_utils.js

```diff
@@ -1,58 +1,22 @@
 /**
  * Provides utilities for validating libraries during creation.
  */
-define(["jquery", "underscore", "gettext", "js/views/utils/view_utils"],
-    function ($, _, gettext, ViewUtils) {
+define(["jquery", "underscore", "gettext", "js/views/utils/view_utils", "js/views/utils/create_utils_base"],
+    function ($, _, gettext, ViewUtils, CreateUtilsFactory) {
         "use strict";
-        return function (selectors, classes) {
-            var validateTotalKeyLength, setNewLibraryFieldInErr, hasInvalidRequiredFields,
-                createLibrary, validateFilledFields, configureHandlers;
+        var CreateLibrary = function (selectors, classes) {
+            CreateUtilsFactory.apply(this);
```

Comment:
Thanks. I've been in Backbone-land so long I've forgotten some of these basic JavaScript things... if I ever truly understood them.